src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml/UIElementTests/UIElementTests.cs

```diff
@@ -43,7 +43,7 @@ public void When_Native_View()
 			_app.WaitForElement("SpacerBorder");
 			var spacerRect = _app.GetRect("SpacerBorder");
 
-			using var scrn = TakeScreenshot("Ready");
+			using var scrn = TakeScreenshot("Ready", ignoreInSnapshotCompare: true);
```

Comment:
This is inconsistent between runs?

Reply:
Nope... it's just redundant with generated screenshots.